Adds additional clothing slots to aghost, and updates aghost bag contents

SphiraI opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
Adds
- Mask, Neck, Eyes, ID slot to aghost inventory proto
- adds Forensics scanner to aghost bag
- adds Omnitool to aghost bag
- adds Experimental welder to aghost bag
- adds a banana cream pie to aghost bag
- Takes the AdminPDA from the bag, and gives it it's own ID slot on the ghost.
<!-- What did you change in this PR? -->

## Why
Ever since we got a hat slot, admemes have tried their damndest to VV additional slots for style reasons, and have not been able to. So this is a mini-bulk update to give further options, along with adding a couple more common tools that are nice to have on hand.
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->

## Media
![image](https://github.com/space-wizards/space-station-14/assets/145869023/358cabb2-8c50-4f6b-b87e-5816b1e51122)
![image](https://github.com/space-wizards/space-station-14/assets/145869023/4e8a0701-62e4-4d98-b104-1768e390f5c5)


- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl: Sphiral
ADMIN:
- tweak: Aghosts can now smoke, wear glasses, and cloaks! Also, aghost bag contents now have some more commonly needed tools!

EmoGarbage404 commented:
Just give it the basic human inventory at this point because i dont really see the point in keeping a secondary prototype.

deltanedas commented:
remove t-ray too its useless

SphiraI commented:
> Just give it the basic human inventory at this point because i dont really see the point in keeping a secondary prototype.

I'll change it to that if thats what people want, but I'm fairly certain it would just cause unnecessary bloat of the inventory screen, and could cause some... *interesting* sprites, since ghosts don't fit into stuff like jumpsuits.

SphiraI commented:
> remove t-ray too its useless

We have an admin that *really* would rather keep it, so I don't really want to remove it, if I don't have to.

PJB3005 commented:
> could cause some... interesting sprites

With great power comes great responsibility.

SphiraI commented:
> B > show subfloor is a thing i dont know when you would be able to be an aghost but not use that

They know about the sandbox options, but would rather only see the components near them, instead of all over the screen.

VasilisThePikachu commented:
> Mask, Neck, Eyes, ID slot to aghost inventory proto

I'm pretty sure I removed those from my pr since they were not wanted

VasilisThePikachu commented:
> Takes the AdminPDA from the bag, and gives it it's own ID slot on the ghost.
I had also moved it to the bag cause there is less chance for an admin to drop it accidentally

SphiraI commented:
> I'm pretty sure I removed those from my pr since they were not wanted
> I had also moved it to the bag cause there is less chance for an admin to drop it accidentally

- I know we've had plenty of admins wanting masks (mainly for smoking, lol)
- Glasses for quick swap sec/med huds because some admins don't the clutter of having them all on all the time
- Neck slot for various pins or if people want cloaks
- We want to be able to quickly grab/open the PDA, not deal with the BoH menu that takes up a third of our screens @_@ (and technically, the only reason we would actually grab it is to check a door's logs)

VasilisThePikachu commented:
> > I'm pretty sure I removed those from my pr since they were not wanted
> > I had also moved it to the bag cause there is less chance for an admin to drop it accidentally
> 
> * I know we've had plenty of admins wanting masks (mainly for smoking, lol)
> * Glasses for quick swap sec/med huds because some admins don't the clutter of having them all on all the time
> * Neck slot for various pins or if people want cloaks
> * We want to be able to quickly grab/open the PDA, not deal with the BoH menu that takes up a third of our screens @_@ (and technically, the only reason we would actually grab it is to check a door's logs)

2: tbh I would prefer an action but there's already a million of those so idk
4: fun fact it has a 0 second health analyzer

SphiraI commented:
> 2: tbh I would prefer an action but there's already a million of those so idk 4: fun fact it has a 0 second health analyzer

An action would be nice, but ~~oh god the actions bar~~
I would personally say just make all the actions catridges for the PDA but thats for another PR

metalgearsloth commented:
Why are we wasting pvs budget on this? Plus you need to add even more hacks to accommodate mapping

SphiraI commented:
> Why are we wasting pvs budget on this? Plus you need to add even more hacks to accommodate mapping

Not real sure what any of this has to do with mapping.
As for the cosmetic side of things, it's really a small change to let admins have additional personality to their ghosts. I doubt one more entity with half as many slots than a normal humanoid would have much of an impact.

VasilisThePikachu commented:
I'm gonna be honest here sloth. If a little backpack and inventory prototype is considered "waste of budget" then I think the pvs should be looked at.

metalgearsloth commented:
Unless you plan to address the mapping problems + code hacks this will cause this is not going to get merged https://discord.com/channels/310555209753690112/900426319433728030/1207840982385557606

VasilisThePikachu commented:
> > I'm gonna be honest here sloth. If a little backpack and inventory prototype is considered "waste of budget" then I think the pvs should be looked at.
> 
> https://github.com/space-wizards/RobustToolbox/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed+author%3Ametalgearsloth+PVS

Perfect, now some small prototypes are not a problem if any of those prs fix it! I highly doupt some extra processing time by the server is really gonna effect experience that much. And if it is the issue would not be a backpack. It would be the pvs
